src/filters/network.rs

```diff
@@ -3283,17 +3329,11 @@ mod match_tests {
         {
             let filter = r#"/^https?:\/\/([0-9a-z\-]+\.)?(9anime|animeland|animenova|animeplus|animetoon|animewow|gamestorrent|goodanime|gogoanime|igg-games|kimcartoon|memecenter|readcomiconline|toonget|toonova|watchcartoononline)\.[a-z]{2,4}\/(?!([Ee]xternal|[Ii]mages|[Ss]cripts|[Uu]ploads|ac|ajax|assets|combined|content|cov|cover|(img\/bg)|(img\/icon)|inc|jwplayer|player|playlist-cat-rss|static|thumbs|wp-content|wp-includes)\/)(.*)/$image,other,script,~third-party,xmlhttprequest,domain=~animeland.hu"#;
             let network_filter = NetworkFilter::parse(filter, true, Default::default()).unwrap();
-            let regex = Arc::try_unwrap(network_filter.get_regex()).unwrap();
-            assert!(
```

Comment:
Removed because there is no way to get the `CompiledRegex` type.
Do it important here to check it?

Reply:
This test is just checking that the filter has a compiled regex. The equivalent would be checking that an external `RegexManager` compiles the regex for it (after a `matches` call). I don't think it's critical in this test, but we probably should have at least one test somewhere to confirm that the regex count increases as expected.

Reply:
ok, I see.
I've modified the test to check `get_compiled_regex_count()` before/after calling `is_match`.